sourcegraph/sourcegraph
# Fix modal onboarding styles

This PR fixes scope issue in Cody onboarding modal styles. Prior to this fix onboarding styles overrides global modal styles which affects all modals in the app. This PR just tries to fix the scope issue and apply onboarding styles only on onboarding related UI

<img width="1147" alt="Screenshot 2023-12-14 at 11 36 28" src="https://github.com/sourcegraph/sourcegraph/assets/18492575/f1b395eb-91b9-4c5c-b230-0f53e0387f4d">


## Test plan
- Check that Cody onboarding modal still looks correct
- Check that other modals are not affected by these onboarding styles.
